DiscordLink/DiscordDiscordCommands.cs

```diff
@@ -15,18 +19,18 @@ namespace Eco.Plugins.DiscordLink
     public class DiscordDiscordCommands
     {
         public static DiscordColor EmbedColor = DiscordColor.Green;
-        
+
```

Comment:
Hmm... I don't even see these whitespace changes when I'm reviewing the staged file...